Phindr3D-Python/src/Data/Metadata.py

```diff
@@ -86,6 +91,43 @@ def metadataFileExists(self, omf):
             return False
         # else
 
+    def loadMetadataFile(self):
+        filepath = self.metadataFilename
```

Comment:
Check if filepath is not None, is a string, and that the file exists before using it.